.github/workflows/release.yml

```diff
@@ -92,17 +89,12 @@ jobs:
       -
         name: APT Install
         id: aptInstall
-        run: sudo apt-get -y install build-essential debhelper fakeroot
+        run: sudo apt-get -y install build-essential fakeroot
```

Comment:
Do you know if we still need build-essential and fakeroot? Those were only necessary for the `make debian` step.